.gitignore

```diff
@@ -0,0 +1,25 @@
+# Compiled source #
+###################
+*.class
+
+# Packages #
+############
+*.jar
+*.war
+
+# Logs and databases #
+######################
+*.log
+
+# Gradle Files #
+################
+.gradle
+
+# Excluding gradle build #
+##########################
+!/build.gradle
+
```

Comment:
And add `!/gradle/wrapper/gradle-wrapper.jar` to not ignore it somehow

Reply:
Why then it is recommended to add all jar files to .gitignore?

Reply:
Because there are archives(.jar/.war) created by application. We can simply run gradle build/maven build to create these archives, so no need to store them in the repository